### DISCUSSION TITLE
Correct way to setup Virtual Static Groups

### DISCUSSION THREAD

**Discussion by ThomasRasmussen** [OpenIdentityPlatform/OpenDJ#258] [Q&A]
Hi, we're trying to migrate away from OpenDJ 2.6.0 to OpenDJ Community 4.5, but we have issues with the virtual-static-groups which causes errors as they have been defined with MemberURL which include a search for the isMemberOf attribute. 

I have tried to follow the pointers in this issue: https://github.com/OpenIdentityPlatform/OpenDJ/issues/10

But I'm not able to get this to work when searching for members in a group. 

What we are trying to accomplish is:
* Create all our users in the ou=People
* Create teams in ou=Teams, each team has a number of uniqueMember which points to the dn of users in ou=People
* Create virtual static groups in ou=Groups which should then use the Teams for listing members of the groups. Thus allowing us to give access to different groups by specifying teams.

In OpenDJ 2.6.0 it works with specifying the MemberURL search it like this: (&(objectClass=person)(IsMemberOf=cn=Team1,ou=Teams,dc=example,dc=com))

but this fails in OpenDJ 4.5.1

**Comment by vharseko** [comment]
use
```
(&(objectClass=person)(MemberOf=cn=Team1,ou=Teams,dc=example,dc=com))
```
